tests/graphql/02-root-queries.t

```diff
@@ -0,0 +1,224 @@
+        }
+    ],
+    "taskProxy": {
+        "id": "${USER}${ID_DELIM}${SUITE_NAME}${ID_DELIM}20190101T00${ID_DELIM}foo"
+    },
+    "taskProxies": [
+        {
+            "id": "${USER}${ID_DELIM}${SUITE_NAME}${ID_DELIM}20190101T00${ID_DELIM}baa"
+        },
+        {
+            "id": "${USER}${ID_DELIM}${SUITE_NAME}${ID_DELIM}20190101T00${ID_DELIM}bar"
+        },
+        {
+            "id": "${USER}${ID_DELIM}${SUITE_NAME}${ID_DELIM}20190101T00${ID_DELIM}foo"
+        },
+        {
+            "id": "${USER}${ID_DELIM}${SUITE_NAME}${ID_DELIM}20190101T00${ID_DELIM}qar"
+        },
+        {
+            "id": "${USER}${ID_DELIM}${SUITE_NAME}${ID_DELIM}20190101T00${ID_DELIM}qaz"
+        },
+        {
+            "id": "${USER}${ID_DELIM}${SUITE_NAME}${ID_DELIM}20190101T00${ID_DELIM}qux"
+        }
+    ],
+    "family": {
+        "id": "${USER}${ID_DELIM}${SUITE_NAME}${ID_DELIM}FAM"
+    },
+    "families": [
+        {
+            "id": "${USER}${ID_DELIM}${SUITE_NAME}${ID_DELIM}FAM"
+        },
+        {
+            "id": "${USER}${ID_DELIM}${SUITE_NAME}${ID_DELIM}FAM2"
+        },
+        {
+            "id": "${USER}${ID_DELIM}${SUITE_NAME}${ID_DELIM}FAM3"
+        },
+        {
+            "id": "${USER}${ID_DELIM}${SUITE_NAME}${ID_DELIM}FAM4"
+        },
+        {
+            "id": "${USER}${ID_DELIM}${SUITE_NAME}${ID_DELIM}FAM5"
+        },
+        {
+            "id": "${USER}${ID_DELIM}${SUITE_NAME}${ID_DELIM}root"
+        }
+    ],
+    "familyProxy": {
+        "id": "${USER}${ID_DELIM}${SUITE_NAME}${ID_DELIM}20190101T00${ID_DELIM}FAM"
+    },
+    "familyProxies": [
+        {
+            "id": "${USER}${ID_DELIM}${SUITE_NAME}${ID_DELIM}20190101T00${ID_DELIM}FAM2"
+        }
+    ],
+    "edges": [
+        {
+            "id": "${USER}${ID_DELIM}${SUITE_NAME}${ID_DELIM}baa.20190101T00${ID_DELIM}NoTargetNode"
+        },
+        {
+            "id": "${USER}${ID_DELIM}${SUITE_NAME}${ID_DELIM}baa.20190101T00${ID_DELIM}qaz.20190101T00"
+        },
+        {
+            "id": "${USER}${ID_DELIM}${SUITE_NAME}${ID_DELIM}foo.20190101T00${ID_DELIM}NoTargetNode"
+        },
+        {
+            "id": "${USER}${ID_DELIM}${SUITE_NAME}${ID_DELIM}foo.20190101T00${ID_DELIM}bar.20190101T00"
+        },
+        {
+            "id": "${USER}${ID_DELIM}${SUITE_NAME}${ID_DELIM}qar.20190101T00${ID_DELIM}NoTargetNode"
+        },
+        {
+            "id": "${USER}${ID_DELIM}${SUITE_NAME}${ID_DELIM}qux.20190101T00${ID_DELIM}NoTargetNode"
+        },
+        {
+            "id": "${USER}${ID_DELIM}${SUITE_NAME}${ID_DELIM}qux.20190101T00${ID_DELIM}bar.20190101T00"
+        },
+        {
+            "id": "${USER}${ID_DELIM}${SUITE_NAME}${ID_DELIM}qux.20190101T00${ID_DELIM}qaz.20190101T00"
+        }
+    ]
+}
+__HERE__
+cmp_json "${TEST_NAME}-out" \
+    "${TEST_NAME_BASE}-root-queries.stdout" \
+    "$(cat expected)"
+
+purge_suite "${SUITE_NAME}"
+
+exit
```

Comment:
You may want to pass this file through `shellcheck` for better shell script style. (We already do this on Travis CI for shell scripts outside of the `tests/` hierarchy.)

Reply:
Done.